db/sql/migrations/memory_tables.sql

```diff
@@ -44,8 +44,8 @@ CREATE TABLE IF NOT EXISTS "mem_accounts"(
   "blockId" VARCHAR(20),
```

Comment:
That migration file is not used at all, real one is `20160724114255_create_memory_tables.sql`, but ofc cannot be changed.

Reply:
I think we should remove this migration. It was intended to be a seed, rather than migration, but in the end we discovered on rebuild our tables were being purged, not dropped, therefore this migration not used. But this would be a separate issue to address.